.gitignore

```diff
@@ -31,3 +31,5 @@ DerivedDataCache
 *.ipa
 *.zip
 test_results/
+
+Sources/WalletConnectModal/Secrets/secrets.json
```

Comment:
When it used? Other team also need to have this file locally?

Reply:
Only needed if you want to run WalletConnectModal previews. If you don't have it, you will get a somewhat explanatory error to create it.